hack/run_e2e_test.sh

```diff
@@ -30,6 +30,7 @@ check-env "FSS_VOLUME_HANDLE"         $FSS_VOLUME_HANDLE
 check-env "MNT_TARGET_ID"             $MNT_TARGET_ID
 check-env "MNT_TARGET_SUBNET_ID"      $MNT_TARGET_SUBNET_ID
 check-env "MNT_TARGET_COMPARTMENT_ID" $MNT_TARGET_COMPARTMENT_ID
+check-env "ENABLE_PARALLEL_RUN"       $ENABLE_PARALLEL_RUN
```

Comment:
ginkgo should have the -p parameter to enable parallel runs. Please check the changes to this file once again.

Reply:
Corrected this.